### ISSUE TITLE
[Picker] The FieldButton needs .is-focused classing.

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-css#1080]
## Description
An `.is-focused` mirror of the `:focus-visible` styles should be shipped for `Picker` to match the same in `FieldButton`.

## Why do you need this feature or component?
A `FieldButton` is the basis for the styling (particularly `:focus-visible`) of the `Picker` pattern. When paired with a `<label>` the accessibility team would like to _always_ (which is not currently true of any `:focus-visible` heuristic) apply the focus styling to a `Picker`. 

## Additional context
Roughly:
```
<label class="spectrum-FieldLabel">Click me...</label>
<div class="spectrum-Picker">
    <button class="spectrum-FieldButton">Then I'll be focused both functionally and visually!</button>
</div>
```

**Comment by lazd** [comment]
FieldButton has ben killed. RIP FieldButton. Picker already has a `.is-focused` class. Does this do the deed? https://github.com/adobe/spectrum-css/blob/main/components/picker/index.css#L189-L198

**Comment by Westbrook** [comment]
Yep, yep, cool! Any info on how much more work is scheduled before the next beta release?

**Comment by lazd** [comment]
I think we'll fix that backtick issue then do the release!

**Comment by lazd** [comment]
Looks like we're good here, gonna go ahead and close this. Thanks!
